<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>book</class>
 <widget class="QDialog" name="book">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Tytuł</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="titleEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="authorLabel">
     <property name="text">
      <string>Autor</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="authorEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="releaseDateLabel">
     <property name="text">
      <string>Data wydania</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateEdit" name="dateEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="bookIdLabel">
     <property name="text">
      <string>Numer książki: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="confirmButton">
     <property name="text">
      <string>Zatwierdź</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Anuluj</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
